### DISCUSSION TITLE
Independent release:     O Yes      O No

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#100] [General]
I regularly create my samplesets with 2 separate releases (for staccato and portato) and the original release within the main file for the longest release.

What should be the correct setting for "independent releases"?

Regards
Mark

**Comment by larspalo** [comment]
The short answer is none. Watch https://youtu.be/Dkt8u6Zd8Vw?feature=shared for information about the independent release. Independent release is used together with percussive samples to more efficently create noise effects than we used to do earlier.

For the normal releases I recommend that one use separate directories within the main stop/rank directory for them. Also if one names the folders to something like _rel00150_ or _rel00550_ the naming can easily be used to identify the releases while loading the samples and also set a default max key press time for them from the number.
